pmoves/tests/smoke/test_nats_authentication.py

```
@@ -108,78 +121,63 @@ def test_env_shared_still_has_nats_user_and_password() -> None:
-
-    client = docker.from_env()
-
-    # Get NATS container
     try:
-        nats_container = client.containers.get("pmoves-nats-1")
-
-        # Check NATS configuration - it should require authentication
-        # NATS with --auth flag or NATS_USER/NATS_PASSWORD environment variables
-        result = nats_container.exec_run("env | grep -i nats")
+        result = subprocess.run(
+            ["docker", "inspect", "pmoves-nats-1", "--format", "{{json .Config.Cmd}}"],
```

Ruff (0.15.5)

[error] Starting a process with a partial executable path

(S607)